snap/snapcraft.yaml

```diff
@@ -10,9 +10,16 @@ description: |
 grade: stable
 confinement: strict
 
+slots:
+  # for GtkApplication registration
+  snapd-desktop-integration:
+    interface: dbus
+    bus: session
+    name: io.snapcraft.desktop-integration
```

Comment:
This should be io.snapcraft.SnapDesktopIntegration

Reply:
Absolutely right.

Reply:
Fixed.